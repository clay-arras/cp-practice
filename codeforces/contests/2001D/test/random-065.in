1
5
13 13 5 18 10
