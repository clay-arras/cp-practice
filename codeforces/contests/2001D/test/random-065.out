4
13 5 18 10
